pages/issues/[id].tsx

```diff
@@ -71,36 +71,50 @@ export default function IssueComponent({ issueData }: { issueData: Issue }): JSX
           </Text>
           <SocialShare url={`${siteConfig.url}${router.asPath}`} title={issueData.meta.title} />
         </div>
-        <IssueItem id="tip" title="Tip of the day" icon={<TipIcon />}>
-          <Text type="base" additionalStyles="py-4 relative z-10">
-            <Markdown>{issueData.tipOfTheWeek.desc}</Markdown>
-          </Text>
-          <CodeSnippet snippet={issueData.tipOfTheWeek.snippet} />
-        </IssueItem>
-        <IssueItem id="articles" title="Articles" icon={<ArticlesIcon />}>
-          {issueData.articles.map(article => (
-            <ArticleItem article={article} key={article.url} />
-          ))}
-        </IssueItem>
-        <IssueItem id="tools" title="Tools" icon={<ToolsAndResourcesIcon />}>
-          {issueData.tools.map(tool => (
-            <ToolItem tool={tool} key={tool.url} />
-          ))}
-        </IssueItem>
-        <IssueItem id="dev" title="Dev Of The Week" icon={<DevOfTheWeekIcon />}>
-          <DevOfTheWeekItem devOfTheWeek={issueData.devOfTheWeek} />
-        </IssueItem>
-        <IssueItem id="talk" title="Tech talks" icon={<TechTalksIcon />}>
-          {issueData.talks.map(talk => (
-            <TechTalk key={talk.talkURL} techTalk={talk} />
-          ))}
-        </IssueItem>
-        <IssueItem id="quiz" title="Quiz" icon={<QuizIcon />}>
-          <Quiz quiz={issueData.quiz} />
-        </IssueItem>
-        <IssueItem id="gif" title="This Week in GIF" icon={<GifIcon />}>
-          <GIFItem gif={issueData.gif} />
-        </IssueItem>
+        {issueData?.tipOfTheWeek !== null ? (
+          <IssueItem id="tip" title="Tip of the day" icon={<TipIcon />}>
+            <Text type="base" additionalStyles="py-4 relative z-10">
+              <Markdown>{issueData.tipOfTheWeek.desc}</Markdown>
+            </Text>
+            <CodeSnippet snippet={issueData.tipOfTheWeek.snippet} />
+          </IssueItem>
+        ) : null}
```

Comment:
Don't think we need the ternary operator here can do it like this maybe?

```suggestion
        {issueData?.tipOfTheWeek && (
          <IssueItem id="tip" title="Tip of the day" icon={<TipIcon />}>
            <Text type="base" additionalStyles="py-4 relative z-10">
              <Markdown>{issueData.tipOfTheWeek.desc}</Markdown>
            </Text>
            <CodeSnippet snippet={issueData.tipOfTheWeek.snippet} />
          </IssueItem>
        )}
```

Reply:
I get your point but I usually avoid using `&&` operator for conditional rendering.

Read [Use ternaries rather than && in JSX](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx) by @kentcdodds for reference.

Reply:
Okay makes sense 👌